docs/rule/no-dom-events.md

```diff
@@ -0,0 +1,27 @@
+## no-dom-events
```

Comment:
I'd also like it if this rule pointed out some of the other pitfalls to using `onclick=` style. One common one is that it doesn't work for svg (since there is no `onclick` property of `SVGElement`)...

Reply:
Added a list with that example.